assets/src/edit-story/components/canvas/carousel/compactIndicator.js

```diff
@@ -27,14 +27,18 @@ import { forwardRef } from 'react';
  */
 import { COMPACT_THUMB_WIDTH, COMPACT_THUMB_HEIGHT } from '../layout';
 
-function CompactIndicatorWithRef({ onClick, isActive, ariaLabel, role }, ref) {
+function CompactIndicatorWithRef(
+  { onClick, isActive, ariaLabel, role, tabIndex },
```

Comment:
With the renaming of `ariaLabel` to `aria-label` in `<Carousel />` (as mentioned in the previous comment), you can completely delete this component altogether and just make `Indicator` default export directly!

This component has zero function anymore, that the simple `styled.button` doesn't also have. The sole purpose right now is basically to rename the property, but without that purpose (which is silly anyway), it really doesn't do a thing.

That'll significantly reduce the complexity of this file - all the forward-ref and proptype-stuff can just go out the door!